plugins/include/sqlx.inc

```diff
@@ -105,171 +129,248 @@ native SQL_QuoteString(Handle:db, buffer[], buflen, const string[]);
+ *       Each time the result set from the previous call will be freed.
+ *
+ * @param query         Handle of a prepared query to be executed.
+ * 
+ * @return              1 if the query succeeded, 0 if the query failed.
  */
 native SQL_Execute(Handle:query);
 
+
 /**
  * Gets information about a failed query error.
- * Returns the errorcode.
+ *
+ * @param query         Handle of a query to extract the error from.
+ * @param error         Buffer where to store the error string.
+ * @param maxlength     The maximum length of the output buffer.
+ * 
+ * @return              The error code.
  */
 native SQL_QueryError(Handle:query, error[], maxlength);
 
 
 /**
- * Returns 1 if there are more results to be read,
- *  0 otherwise.
+ * Check whether there are more results to be read.
+ *
+ * @param query         Handle of a query to check.
+ * 
+ * @return              1 if there are more results, 0 otherwise.
  */
 native SQL_MoreResults(Handle:query);
 
 
 /**
- * Tells whether a specific column in the current row
- *  is NULL or not.
+ * Tells whether a specific column in the current row is NULL or not.
+ *
+ * @param query         Handle of a query to check.
+ * @param column        Which column to check for NULL.
+ * 
+ * @return              1 if the column is NULL, 0 otherwise.
  */
 native SQL_IsNull(Handle:query, column);
 
+
 /**
  * Retrieves the current result.
- * A successful query starts at the first result,
- *  so you should not call SQL_NextRow() first.
- * Passing no extra params - return int
- * Passing one extra param - return float in 1st extra arg
- * Passing two extra params - return string in 1st arg, max length in 2nd
- * Example:
- *  new num = SQL_ReadResult(query, 0)
- *  new Float:num2
- *  new str[32]
- *  SQL_ReadResult(query, 1, num2)
- *  SQL_ReadResult(query, 2, str, 31)
+ * A successful query starts at the first result, so you should not call
+ * SQL_NextRow() first.
+ * 
+ * @note Example how to get different types of values:
+ *       new num = SQL_ReadResult(query, 0)
+ *       new Float:num2
+ *       new string[32]
+ *       SQL_ReadResult(query, 1, num2)
+ *       SQL_ReadResult(query, 2, str, 31)
+ *
+ * @param query         Handle of a query to read results from.
+ * @param column        Which column to get the value from.
+ * @param ...           Passing no extra arguments - returns an integer.
+ *                      Passing one extra argument - returns a float in the first extra argument
+ *                      Passing two extra params - returns a string in the first argument
+ *                      with a maximum string length in the second argument.
+ * 
+ * @return              If no extra arguments are passed, returns an integer value.
  */
 native SQL_ReadResult(Handle:query, column, any:...);
 
 
 /**
- * Advances to the next result (return value should be ignored).
+ * Advances to the next result (row).
+ *
+ * @param query         Handle of a query.
+ * 
+ * @noreturn
```

Comment:
Misses `@error` with those possibles errors `Invalid query handle`, `No result set in this query`